client/app/bundles/Comments/components/SimpleCommentScreen.jsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import Immutable from 'immutable';
 import request from 'axios';
+import _ from 'lodash';
+import pureRender from 'pure-render-decorator';
+
 import CommentForm from './CommentForm';
 import CommentList from './CommentList';
 import metaTagsManager from '../utils/metaTagsManager';
-import _ from 'lodash';
 
+@pureRender
 class SimpleCommentScreen extends React.Component {
   constructor(props, context) {
     super(props, context);
     this.state = {
       $$comments: Immutable.fromJS([]),
-      ajaxSending: false,
+      isSaving: false,
```

Comment:
Warning: Failed propType: Required prop `isSaving` was not specified in `CommentForm`. Check the render method of `SimpleCommentScreen`. And this kept the button enabled while saving...